h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -179,27 +192,39 @@
     final double rows_per_chunk = chunk_size; //why not numRow * chunk_size / totSize; it's supposed to be rows per chunk, not the byte size
     final int num_chunks = Vec.nChunksFor(numRow, (int) Math.ceil(Math.log1p(rows_per_chunk)), false);
 
-    if (optimize) {
+    if (streaming && optimize) {
+      Log.warn("Parameter 'optimize' will be ignored in streaming ingest.");
+    }
+    final boolean estimateConcurrentConnections = optimize && (! streaming);
+    if (estimateConcurrentConnections) {
+      assert ! streaming;
```

Comment:
a bit redundant, or processor is really broken

Reply:
yep, agree - can be removed - it is more of a leftover than an intention